### PR TITLE
timestamp response creation implemented

### PR DESCRIPTION
**Pull request by bilalashraf123**

### CODE REVIEW

File: timestamp.go [hunk 1]
```diff
@@ -154,8 +175,12 @@ type Timestamp struct {
 	HashedMessage []byte
 
 	Time         time.Time
-	Accuracy     time.Duration
+	Accuracy     Accuracy
```
Comment (vanbroup) [inline timestamp.go]: Please keep the accuracy as `time.Duration` this can be calculated as following: https://play.golang.org/p/9HdBIdYJ4Pd

Comment (bilalashraf123) [reply timestamp.go]: Done.

File: timestamp.go [hunk 2]
```diff
@@ -302,6 +337,201 @@ func CreateRequest(r io.Reader, opts *RequestOptions) ([]byte, error) {
...
+	return signingCertV2Bytes, nil
+}
+
+func generateSignedData(tstInfo []byte, privateKey crypto.PrivateKey, certificate *x509.Certificate) ([]byte, error) {
+	signedData, err := pkcs7.NewSignedData(tstInfo)
+	if err != nil {
+		return nil, err
+	}
+	signedData.SetDigestAlgorithm(pkcs7.OIDDigestAlgorithmSHA256)
+	// signedData.SetEncryptionAlgorithm(pkcs7.OIDEncryptionAlgorithmRSA)
```
Comment (vanbroup) [inline timestamp.go]: I suppose this comment can be removed?

File: timestamp_test.go [hunk 3]
```diff
@@ -143,9 +148,10 @@ func TestParseResponse(t *testing.T) {
 				t.Errorf("resp.Time: got %v, want %v", resp.Time, tc.time)
 			}
 
-			if resp.Accuracy != tc.accuracy {
-				t.Errorf("resp.Accuracy: got %v, want %v", resp.Accuracy, tc.accuracy)
-			}
+			// FIXME:
```
Comment (vanbroup) [inline timestamp_test.go]: Can this be fixed?

File: timestamp_test.go [hunk 4]
```diff
@@ -176,9 +182,10 @@ func TestParse(t *testing.T) {
 		t.Errorf("ts.Time: got %v, want %v", ts.Time, tsTime)
 	}
 
-	if ts.Accuracy != time.Second {
-		t.Errorf("ts.Accuracy: got %v, want %v", ts.Accuracy, time.Second)
-	}
+	// FIXME:
```
Comment (vanbroup) [inline timestamp_test.go]: Same here?

File: timestamp.go (hunk 2 above)
Comment (bilalashraf123) [reply timestamp.go]: Done.

File: timestamp_test.go (hunk 3 above)
Comment (bilalashraf123) [reply timestamp_test.go]: Done.

File: timestamp_test.go (hunk 4 above)
Comment (bilalashraf123) [reply timestamp_test.go]: Done.

**Pull request merged**